File: Python/1769_minimum_number_of_operations_to_move_all_balls_to_each_box.py
class Solution:
    def minOperations(self, boxes: str) -> list[int]:
        ret_list = []
        operations = 0
        for i in range(len(boxes)):
            for j, trial_box in enumerate(boxes):
                if trial_box == "1":
                    operations += abs(i - j)
            ret_list.append(operations)
            operations = 0
        return ret_list

File: JavaScript/1769_minimum_number_of_operations_to_move_all_balls_to_each_box.js
/**
 * @param {string} boxes
 * @return {number[]}
 */
var minOperations = function(boxes) {
    let ans = new Array(boxes.length).fill(0);
    for (let i = 0; i < boxes.length; i++) {
        for (const [j, box] of boxes.split('').entries()) {
            if (box == '1') {
                ans[i] += Math.abs(i - j);
            }
        }
    }
    return ans;
};

